### PR TITLE
[PATCH]Make code generator window draggable

### PR DESCRIPTION
**Pull request by akashdsouza**
https://github.com/QuantumInformation/test-recorder/issues/4

### CODE REVIEW

**Comment by quantuminformation** [comment]
Many thanks, I'll test this tonight!

**Comment by quantuminformation** [comment]
VanillaJS too nice!

File: src/TestRecorder.ts [hunk 1]
```diff
@@ -170,6 +171,30 @@ export class TestRecorder {
     })
   }
 
+  setUpDragAndDrop() {
+    let testRecorderUI = document.getElementById('testRecorderUI')
```
Comment (quantuminformation) [inline src/TestRecorder.ts]: please use `this.hostElement` for consistency

Comment (akashdsouza) [reply src/TestRecorder.ts]: Done.

Comment (quantuminformation) [reply src/TestRecorder.ts]: please push

Comment (akashdsouza) [reply src/TestRecorder.ts]: I have pushed already :confused:

**Comment by quantuminformation** [comment]
Did you try it with the test app? looks weird:

image
image

**Comment by akashdsouza** [comment]
It seemed fine when I was working. I'm out now. Perhaps we can discuss this later on slack?

**Comment by quantuminformation** [comment]
Can you come in here: https://discord.gg/aYQcDs8 I have a room for this repo. Thanks

**Comment by akashdsouza** [comment]
I have made the changes discussed and removed the css changes done earlier.  I still haven't been able to test this on another machine.

**Review by quantuminformation — APPROVED** [review]
great work

**Pull request merged**